### ISSUE TITLE
No data? Subscription doesn't send "ready" and blocks all other subscriptions 

### ISSUE THREAD

**Issue by DenisGorbachev** [numtel/meteor-pg#4]
```
Meteor.publish("Avatars", (api) ->
  check(api, String)
  return [] unless @userId
  PgConnection.select('SELECT a."id", a."name" FROM "Avatars" AS a WHERE a."api" = $1 AND a."userId" = $2', [api, @userId])
)
```

If there are no avatars for current user, subscription breaks itself and other subscriptions that hit the server after that.

**Comment by numtel** [comment]
Thanks for the issue! Big time whoops on that :hushed: :speak_no_evil: 

Just released, version 0.0.5 fixes this problem.

**Comment by DenisGorbachev** [comment]
Thank you for the fix!
